### ISSUE TITLE
Parameters in flags.go are not respected

### ISSUE THREAD

**Issue by Varunram** [Musicoin/desktop#254]
Adding extra parameters (e.g. `--port`) to config/config.std.js are not respected while starting the gmc client. They should be.

**Comment by Varunram** [comment]
More info on this: We create a file at `$HOME/config/` from which the wallet reads the config from. This is good, but this prevents the user from actually changing stuff within `/config` and then testing it. If we are to improve it in some way, we must either link both the files or have a `musicoin.conf` feel as suggested in https://github.com/Musicoin/desktop-interface/issues/8

**Comment by etaletai13** [comment]
Let's go ahead and close this with preference to Musicoin/desktop-interface#8 because the full solution is in filling that one.

**Comment by Varunram** [comment]
Yeah, I guess that would be nice, but this is a more popular watched repo than the other one, so makes sense having this as well.

**Comment by etaletai13** [comment]
Noted and agreed
